bolt-design-system/bolt
# feat: expose props to edit the image atop the interactive pathways co…

## Asana

https://app.asana.com/0/1126340469288208/1139491913875001/f

## Summary

The image at the top of the pathways component (two diamond logo) can now be changed. the chosen solution was to expose props to control this component through the pathways component, reducing the ability for an editor to break layout.

## Details

During an editing session, an editor can now select the interactive pathways component and use the editor interface to edit the image at the top of the interactive pathways.

## How to test

- Go to http://localhost:3000/pattern-lab/patterns/06-experiments-micro-journeys-combos--92-micro-journey-in-themes/06-experiments-micro-journeys-combos--92-micro-journey-in-themes.html
- Start an editing session
- highlight the interactive pathways component, either by clicking on it or selecting in the layers panel
- the editor should now show two fields, one for the image src and one for the alt text. The default (if no value is provided) is the two diamond image. Try adding a different url https://www.fillmurray.com/g/200/200
